charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -212,11 +212,24 @@ spec:
         image: budibase/apps:{{ .Values.globals.appVersion }}
         imagePullPolicy: Always
         livenessProbe:
+          httpGet:
+            path: /health
+            port: {{ .Values.services.apps.port }}
+          initialDelaySeconds: 10
```

Comment:
Is this too long? A pod will need at least 10 seconds to start essentially? Would `5` work here even?

Reply:
This is for the liveness, which affects only killing a container after it's started. 
The readinessprobe below is what determines when to start sending traffic. I intentionally made the delay of the liveness to be longer than the readiness, which is set to 5

Reply:
super, all good then!